site/user-guide.md

```diff
@@ -98,7 +98,9 @@ Oracle recommends careful consideration before deciding to expose any administra
 * Flannel networking v0.9.1-amd64 (check with `docker images | grep flannel`).
 * Docker 18.03.1.ce (check with `docker version`).
 * Helm 2.8.2+ (check with `helm version`).
-* Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 (check with `docker run store/oracle/weblogic:12.2.1.3 /u01/oracle/OPatch/opatch lsinventory`).
+* Oracle WebLogic Server 12.2.1.3.0 with patch 29135930.
+   * Check WLS version with `docker run store/oracle/weblogic:12.2.1.3 sh -c` `'source $ORACLE_HOME/wlserver/server/bin/setWLSEnv.sh > /dev/null 2>&1 && java weblogic.version'`.
+   * Check WLS patches with `docker run store/oracle/weblogic:12.2.1.3 sh -c` `'$ORACLE_HOME/OPatch/opatch lspatches'`.
    * The existing WebLogic Docker image, `store/oracle/weblogic:12.2.1.3`,
 was updated on January 17, 2019, and has all the necessary patches applied.
    * A `docker pull` is required if you pulled the image prior to that date.
```

Comment:
Maybe make the last two bullets the first two bullets - so the references to 'store/oracle/weblogic:12.2.1.3' in the 'Check' bullets have more context.  Otherwise, LGTM.

Reply:
I've verified the syntax - cut/pasting from the `rich diff`, then running the command, worked fine...

Reply:
@rjeberhard I think it's ready now to be merged.